deploy/crds/apps.3scale.net_apimanagerbackups_crd.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: apimanagerbackups.apps.3scale.net
+spec:
+  group: apps.3scale.net
+  names:
+    kind: APIManagerBackup
+    listKind: APIManagerBackupList
+    plural: apimanagerbackups
+    singular: apimanagerbackup
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: APIManagerBackup is the Schema for the apimanagerbackups API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: APIManagerBackupSpec defines the desired state of APIManagerBackup
+          properties:
+            apiManagerName:
```

Comment:
The purpose is that currently there are two methods to retrieve the APIManager to backup:
1. You provide the APIManager name that you want to backup through this 'apiManagerName' optional attribute and the operator tries to find that name.
1. In case apiManagerName is not provided it tries to find an existing APIManager in the namespace where the 3scale-operator has been deployed

Reply:
Only one apimanager object per namespace is allowed.

Reply:
I added that option just as an additional possibility where the user could made that explicit on the spec. I understand you don't see it necessary and you want it removed. I don't see it adds a lot of value so I have removed it